README.md

```diff
@@ -823,6 +824,10 @@ module.exports = function(app) {
 };
 ```
 
+# Partitioned database
+
+Refer to https://github.com/strongloop/loopback-connector-cloudant/blob/master/doc/partitioned-db.md
```

Comment:
maybe add some more annotation around it? Suggestion:
```
If you're using partitioned database, see details on how to configure the connector: ...
```

Reply:
👍